apps/crn-server/test/controllers/news.test.ts

```diff
@@ -120,6 +120,164 @@ describe('News controller', () => {
 
       expect(result.items[0]!.thumbnail).toBeUndefined();
     });
+
+    test('Should query data properly when request does not have options', async () => {
+      nock(baseUrl)
+        .get(`/api/content/${appName}/news-and-events`)
+        .query({
+          q: JSON.stringify({
+            take: 8,
+            filter: { path: 'data.type.iv', op: 'ne', value: 'Tutorial' },
+            sort: [{ order: 'descending', path: 'created' }],
+          }),
+        })
+        .reply(200, newsSquidexApiResponse);
+
+      const result = await news.fetch();
+
+      expect(result).toEqual(listNewsResponse);
+    });
+
+    describe('Frequency Filter', () => {
+      test('Should query data properly when News Articles is selected', async () => {
+        nock(baseUrl)
+          .get(`/api/content/${appName}/news-and-events`)
+          .query({
+            q: JSON.stringify({
+              take: 8,
+              skip: 5,
+              filter: {
+                and: [
+                  { path: 'data.type.iv', op: 'ne', value: 'Tutorial' },
+                  {
+                    or: [
+                      {
+                        path: 'data.frequency.iv',
+                        op: 'empty',
+                        value: null,
+                      },
+                      {
+                        path: 'data.frequency.iv',
+                        op: 'in',
+                        value: ['CRN Quarterly', 'News Articles'],
+                      },
+                    ],
+                  },
+                ],
+              },
+              sort: [{ order: 'descending', path: 'created' }],
+            }),
+          })
+          .reply(200, newsSquidexApiResponse);
+
+        const result = await news.fetch({
+          take: 8,
+          skip: 5,
+          filter: {
+            frequency: ['CRN Quarterly', 'News Articles'],
+          },
+        });
+
+        expect(result).toEqual(listNewsResponse);
+      });
+
+      test('Should query data properly when News Articles is not selected', async () => {
+        nock(baseUrl)
+          .get(`/api/content/${appName}/news-and-events`)
+          .query({
+            q: JSON.stringify({
+              take: 8,
+              skip: 5,
+              filter: {
+                and: [
+                  { path: 'data.type.iv', op: 'ne', value: 'Tutorial' },
+                  {
+                    path: 'data.frequency.iv',
+                    op: 'in',
+                    value: ['CRN Quarterly'],
+                  },
+                ],
+              },
+              sort: [{ order: 'descending', path: 'created' }],
+            }),
+          })
+          .reply(200, newsSquidexApiResponse);
+
+        const result = await news.fetch({
+          take: 8,
+          skip: 5,
+          filter: {
+            frequency: ['CRN Quarterly'],
```

Comment:
I've changed one record in squidex to have frequency CRN Quaterly then started the app and call http://localhost:3333/news?take=10&skip=0&filter=CRN+Quarterly passing an auth token I've got from the network tab in the hub. It worked 🎉 

<img width="1175" alt="Screen Shot 2022-09-20 at 08 35 04" src="https://user-images.githubusercontent.com/16595804/191466051-ef7135db-aeb6-4d50-9250-16c0f2de6ee8.png">